public/app/core/components/SplitPaneWrapper/SplitPaneWrapper.tsx

```diff
@@ -74,6 +75,7 @@ export class SplitPaneWrapper extends PureComponent<React.PropsWithChildren<Prop
 
     // Limit options pane width to 90% of screen.
     const styles = getStyles(config.theme2, splitVisible);
+    const dragStyles = getDragStyles(config.theme2);
```

Comment:
i think this can/should just be `const dragStyles = useStyles2(getDragStyles)`

Reply:
it's a class component

could convert it to functional in a follow up PR if you want, it doesn't look too tricky

Reply:
ahhhh. true dat